modules/imgproc/src/lsd.cpp

```diff
@@ -75,6 +75,12 @@ struct edge
     bool taken;
 };
 
+struct coorlist
+{
+    cv::Point2i p;
+    int norm;
```

Comment:
Looks like this struct is misnamed ("list" part should be removed after removing "next" entry - something like PointEntry).

Reply:
Thanks for your comment.
I think to change the structure name from "coorlist" to "coorPoint", what do you think?
In addition, since the member variable "list" does not already have a list structure, I'd like to change the name of "list" to "ordered_points", what do you think?

Reply:
"coor" here is a just "typo" for "coordinate". "coordinatePoint" is redundant name.

> ordered_points

Sounds good.